app/assets/stylesheets/modules/master_overlay.css.scss

```diff
@@ -418,13 +418,14 @@ $parent-width: $parent-split * 0.21;
 }
 
 .row {
+  -js-display: flex;
```

Comment:
Besides -js-display, the fixed-width mini map also contributes to the issue in IE11. Therefore, I made changes to .flex-column class as well.